update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -0,0 +1,237 @@
+/*
+This adds corresponding sia405 values for interlis import/export.
+*/
+
+-- Add columns
+
+ALTER TABLE qwat_vl.value_list_base ADD COLUMN sia405code TEXT;
+
+
+-- Populate columns
+
+-- TODO : Bedding material mapping complete [X] to check [ ]
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'aucun';
+UPDATE qwat_vl.bedding SET sia405code = 'Beton' WHERE value_fr = 'béton';
+UPDATE qwat_vl.bedding SET sia405code = 'Fliess' WHERE value_fr = 'géotextile';
+UPDATE qwat_vl.bedding SET sia405code = 'Betonkies' WHERE value_fr = 'gravier';
+UPDATE qwat_vl.bedding SET sia405code = 'Sand' WHERE value_fr = 'sable';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'terre';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cistern SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cover_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- Hydrant material : mapping complete [X] to check [ ]
+UPDATE qwat_vl.hydrant_material SET sia405code = 'Metall' WHERE short_fr LIKE 'F %';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_inf SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_sup SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_output SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_provider SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.leak_cause SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.locationtype SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.nominal_diameter SET sia405code = 'unbekannt' WHERE ???;
```

Comment:
My proposition is 

- to match the NPS value to nominal_diameter (Leitung.Nennweite): e.g. NPS: 1/2 DN: 15 OD: 21.3 -> 1/2
- Leitung.Durchmesser is the DN part
- Leitung.Durchmesser_Aussen is the OD part

Reply:
@olivierdalang Is it already in the python mapping? since diameter, nominal diameter and outer diameter are available in the pipe_material values?

Reply:
I think `qwat_vl.nominal_diameter` is for valves, not for pipes (for which diameters are included in the `pipe_material` value list, and which are already treated in the import script since they are not enums in interlis).

For valves, nominal diameter is already mapped to Nennweite (again not an enum). The issue is that in the interlis model it's limited to 10 characters, so there's a high risk of conveying potentially misleading information (same issue goes for Leitung.Nennweite).

But this is not linked to value lists, so I think we can just skip qwat_vl.nominal_diameter, as there's no matching enum in sia405.